app_dart/lib/src/service/github_service.dart

```diff
@@ -284,4 +284,20 @@ class GithubService {
     final Random rnd = Random();
     return String.fromCharCodes(Iterable<int>.generate(10, (_) => chars.codeUnitAt(rnd.nextInt(chars.length))));
   }
+
+  /// Returns a [Stream] of [Issue]s that match the given [query].
```

Comment:
Are Issues and PRs returned in the same object?

Reply:
Yes, the github package uses the Issue object for both issues and PRs.